tools/query_verification/src/main/java/com/google/bigquery/QueryVerifier.java

```diff
@@ -38,7 +44,48 @@ public void verify() {
     public void verifyDataFree() {
         boolean verificationResult = false;
 
-        // TODO Implement data free verification
+        BigQuery bigQuery = BigQueryOptions.getDefaultInstance().getService();
+
+        List<Table> tables = new ArrayList<Table>();
+
+        // Create tables based on schema
+        if (migratedSchema != null) {
+            try {
+                if (migratedSchema.isInJsonFormat()) {
+                    TableInfo tableInfo = QueryVerifier.getTableInfoFromJsonSchema(migratedSchema);
+                    Table table = bigQuery.create(tableInfo);
+                    tables.add(table);
+                } else {
+                    // TODO Load schema from DDL
+                }
+            } catch (NullPointerException e) {
```

Comment:
Although this saves some lines of code I'd advise against catching a NPE as it could be triggered for a variety of reasons. If possible I'd recommend handling null objects in the verifier code even if it makes a bit more verbose.